ote/src/clj/ote/authorization.clj

```diff
@@ -41,6 +41,11 @@
       :else
       (body-fn))))
 
+(defn is-author?
```

Comment:
A matter of flavour, but should name be something like `user-in-group?`

Reply:
I think that is-author? is really good option here. It separates admin and user level.